src/components/tree.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import {Tree} from '@react95/core';
+import { getTree} from '../Api';
+
+
+function TreeStruture() {
+    const [contentTree,setContentTree] = useState([]);
+    useEffect(() => {
+      const fetchTree = async () => {
+          const contentTree = await getTree();
+          setContentTree(contentTree);
+      } 
+      fetchTree();
+    },[]);
+
+
+  const label = function (str) {
+      return str.substring(str.lastIndexOf('/')+1);
+  }
+
+  const files = contentTree.map(file => {
+    file.label = label(file.path)
+    return file
+  });
+
+
+  function treeify(files) {
+    var path = require('path')
```

Comment:
Is this required to be here. Why can't it be a regular esm import on the top of the file.

if its required then use `const path` not `var`.